### DISCUSSION TITLE
Issue with camera 0x00000004

### DISCUSSION THREAD

**Discussion by Tobias1122334** [jomjol/AI-on-the-edge-device#2765] [Q&A]
Hello,

I tried to install AI on the Edge on an ESP 32 device and get the following error message below.

I flashed the system already a couple of times. I tested the camera with another tool and verirfied that it is working.

AI on the Edge Device
We have one or more critical errors:
0x00000004

Does someone still have an idea how to proceed?

I bought the following device at Amazon:
https://www.amazon.de/dp/B0CJJHXD1W?ref=ppx_yo2ov_dt_b_product_details&th=1

Thanks!

Best regards
Tobias

**Comment by Slider0007** [comment]
This board is not supported by this software out of the box. Camera probably uses other pins than supported ESP32-CAM board. Therefore you get this camera error.

You need to compile by yourself with definition of correct camera pins or buy supported hardware.

**Reply by Tobias1122334** [reply]
Thanks a lot!

**Reply by oferbar** [reply]
Hello,

I have a similar issue with another ESP32-CAM brand: https://www.amazon.com/gp/product/B08P2578LV/

So how can I find the camera correct pinout to be compiled?

Alternatively, where can I find the list of supported devices?

Thanks

**Reply by SybexX** [reply]
https://github.com/jomjol/AI-on-the-edge-device/discussions/1732
https://jomjol.github.io/AI-on-the-edge-device-docs/Error-Codes/
https://jomjol.github.io/AI-on-the-edge-device-docs/Hardware-Compatibility/
